### DISCUSSION TITLE
Custom fields in import and export operation

### DISCUSSION THREAD

**Discussion by alekshw11** [moneymanagerex/moneymanagerex#5417] [Ideas]
Hi folks!
I spent some time in MoneyManagerEx and I really like it.  I was wondering if would be possible to implement custom fields in import and export function from csv or other files. It would be a nice feature for importing data from other places and eventually keeping a complete backup of user data in a text only file. 
I've seen that someone has already asked it on the forum this year but hasn't been any kind of feedback on possible implementation. 
Ready to help if possible.

**Comment by whalley** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/5401
https://github.com/moneymanagerex/moneymanagerex/issues/4270

**Reply by alekshw11** [reply]
Well... Didn't see that.

**Comment by Macaulay1234** [comment]
Good
